Automattic/woocommerce-services
# Update npm-shrinkwrap.json

## Description


Update `npm-shrinkwrap.json` by running `npm run update-deps`.

The direct motivation to do this was to update `@woocommerce/e2e-environment` from `0.2.0` to `0.2.2` as our E2E tests are failing.

**UPDATE:** I previously thought updating `@woocommerce/e2e-environment` will resolve the issue with E2E tests. It seems this isn't the solution and what actually resolves the issue is using a different WP version which is implemented by #2450.

## Testing instructions

(Stolen from @harriswong's [similar PR](https://github.com/Automattic/woocommerce-services/pull/2059).)

1. pull this branch
2. run `npm ci`
3. create an order
4. try to print a label
5. everything should still work.

And to verify E2E tests aren't stuck at "Docker container is still being built" anymore:

1. Run `npm run docker:up`.
2. Run `npm run docker:ping`.

Output should be similar to:

```
Testing URL: http://localhost:8084

Thu Jul  8 10:03:16 CEST 2021 - Docker container is still being built
Thu Jul  8 10:03:26 CEST 2021 - Docker container is still being built
Thu Jul  8 10:03:36 CEST 2021 - Docker container is still being built
Thu Jul  8 10:03:46 CEST 2021 - Docker container had been built successfully
```
